pennylane_qiskit/converter.py

```diff
@@ -157,23 +246,80 @@ def load(quantum_circuit: QuantumCircuit, measurements=None):
         function: the resulting PennyLane template
     """
 
-    # pylint:disable=fixme, too-many-branches
-    def _function(params: dict = None, wires: list = None):
+    def _function(*args, params: dict = None, wires: list = None, **kwargs):
         """Returns a PennyLane template created based on the input QuantumCircuit.
```

Comment:
Should we continue saying "template"? I think this term might have a slightly different connotation now, that this function doesn't really match.

Reply:
changed to "quantum function"